_includes/navbar.html

```diff
@@ -8,102 +8,94 @@
 <nav class="navbar navbar-default">
   <div class="container-fluid">
     <div class="navbar-header">
-      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
+        data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
         <span class="sr-only">Toggle navigation</span>
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
 
-      {% comment %} Select what logo to display. {% endcomment %}
-      <a href="{{ site.esciencecenter_site }}" class="pull-left">
-        <img class="navbar-logo" src="{{ relative_root_path }}{% link /assets/img/nlesc_icon_cyanblack.png %}" alt="Software Carpentry logo" />
+      <a href="{{ site.academy_site }}" class="pull-left"></a>
```

Comment:
we are not using eScience Academy branding anymore. The workshop template already includes the eScience Center logo. Please remove this?

Reply:
Check.